binding.gyp

```diff
@@ -18,6 +18,16 @@
       'dependencies': [
         'deps/libmagic/libmagic.gyp:libmagic',
       ],
+      'conditions': [
+        ['OS=="mac"', {
+          'xcode_settings': {
+            'MACOSX_DEPLOYMENT_TARGET': '10.9',
+            'OTHER_CFLAGS': [
```

Comment:
Instead of using `OTHER_FLAGS`, I think we can just add the following to `xcode_settings`:

```
              'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0',
              'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++1y',  # -std=gnu++1y
              'CLANG_CXX_LIBRARY': 'libc++',
```

Reply:
Confirmed, this works too.